homeassistant/components/pglab/strings.json

```diff
@@ -0,0 +1,27 @@
+{
+  "config": {
+    "step": {
+      "confirm": {
+        "description": "Do you want to set up PG LAB Electronics?"
+      },
+      "config": {
+        "data": {
+          "discovery_prefix": "Discovery topic prefix"
+        }
+      }
```

Comment:
Can this be removed?

Reply:
Thank you for that .... yes I can remove it.